WordPressShared/Core/Views/WPStyleGuide+DynamicType.swift

```diff
@@ -76,8 +76,7 @@ extension WPStyleGuide {
     @objc public class func fontForTextStyle(_ style: UIFont.TextStyle, maximumPointSize: CGFloat = maxFontSize) -> UIFont {
         if #available(iOS 11, *) {
```

Comment:
It looks like WordPressShared targets 11+ now, so we can remove this check.

Reply:
Removed.